apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -174,6 +182,31 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  selectMessage: function(messageId) {
+    var messageNode = this.messageAccessors.find(messageId);
+    messageNode.scriptWith(function(node) {
+      node.scrollIntoView();
+    });
+    messageNode.tap();
+  },
+
+  toggleMessagesSelection: function() {
+    var header = this.accessors.header;
```

Comment:
Can you please file a bug like "Calling element.scrollIntoView moves app out of the screen in b2g-desktop" in gaia::simular bugzilla component that describes steps and probably screenshot and leave comment about this workaround here?